sb/sb_wrapper.py

```diff
@@ -0,0 +1,14 @@
+from common.genesis_wrapper import define_genesis_generator
```

Comment:
We can combine this file with `sb_wrapper_main.py`. See `simple_cb/simple_cb_genesis2.py`. You can name the combined file `sb/sb_genesis.2.py`.